### DISCUSSION TITLE
Mistake on hosting my github page

### DISCUSSION THREAD

**Discussion by Gokul-Raja84** [community/community#64772] [Actions]
### Select Topic Area

General

### Body

I hope this message finds you well. I am currently experiencing critical issues with my GitHub Pages hosting and web apps that I need urgent assistance with.

Unintended CNAME File Creation:
While working on one of my repositories, an unintended CNAME file was created, leading to the automatic creation of an insecure domain-associated repository. I have since removed the CNAME file and deleted the associated repository. However, this action seems to have caused issues across my other hosted web apps.

Web Apps Not Working:
Subsequent to the removal of the unintended CNAME file and repository, my other web apps hosted on GitHub Pages are now showing 404 errors. It appears that the changes I made have affected the functionality of my entire GitHub Pages setup.

I have tried the following steps to resolve the issues:

Checked the repository structure and contents.
Verified that the correct branch is selected for GitHub Pages.
Reviewed settings in the repository and GitHub Pages sections.
Despite these efforts, the issues persist, and I am unable to restore the functionality of my web apps.

I urgently request your assistance in resolving these issues. My GitHub Pages hosting is crucial for my projects, and I am currently unable to provide access to these web apps due to the errors.

Please advise on the best steps to take to restore the proper functioning of my GitHub Pages hosting and web apps. Your prompt attention to this matter is greatly appreciated.

Thank you for your time and support.

**Comment by arafajs** [comment]
- Double check that you have removed any erroneous CNAME files from all your repositories. Any CNAME files can interfere with GitHub Pages functioning correctly.
- Clear your browser cache and test accessing your GitHub Pages sites again. Sometimes cached files can cause issues.
- Check the GitHub Pages section under Settings in each repository. Make sure the correct branch (usually master/main) is selected for GitHub Pages.
- Under Settings > Pages, click the "Change source" dropdown and select "None". Wait a few minutes and change it back to your branch. This may help reset the configuration.
- As a last resort, you can try renaming your publishing branch to something different, wait for Pages to refresh, and then rename it back. This can reset Pages fully.
- If specific repositories are still having issues, rename/delete those repositories and recreate them. This can resolve repository-specific problems.
- Check GitHub Pages status (https://www.githubstatus.com/) for any known outages. These are rare but can happen.

**Reply by Gokul-Raja84** [reply]
Thank you, for your feedback and help. I will make sure to follow your suggestions and I will inform you about the stituation.
